apis/v1alpha2/types.go

```diff
@@ -14,55 +14,32 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package v1alpha1
+package v1alpha2
```

Comment:
My understanding is that you are removing (instead of deprecating) the existing `v1alpha1` version of the API. So this will be a backwards-incompatible change. Are we sure that all clients of this API will also be updated as these changes are rolled out?

Reply:
I think we also need to update the `ProviderConfig` version used in the example `examples/provider.yaml` from `v1alpha1` to `v1alpha2`.

Reply:
1) Yes, we should keep it. Though no v1alpha1 is used in any projects as far as I know. But for compatibility consideration, we should keep it. Reverted the deletion.

2) Updated.

Thanks:)

Reply:
I recalled why I delete the old version. Those two versions depended on the same library `xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"` with different version which could not work in the same repo.

Reply:
Okay then if we do not want to maintain two versions, I assume we will make sure all clients are migrated in parallel to this. We had better make sure this is communicated clearly to all consumers.

Reply:
Yes, confirmed.

I'd like to make a new release based on the latest master code, and it's the final release supporting v1alpha1.